server/src/db-config/models/recipe.ts

```diff
@@ -1,14 +1,14 @@
 import mongoose from "mongoose";
+import { ITag } from "./tag";
+import * as Tag from './tag';
 
 const RecipeSchema = new mongoose.Schema({
     name: { type: String },
     author: { type: String },

```

Comment:
I've added for name but I'm concerned if the type of author shouldn't be User. Otherwise, it will be authorId which should be the always the same length. I prefer to merge it now and if at some point we decide what we need for this attribute to redefined it.